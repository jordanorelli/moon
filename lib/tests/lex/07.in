empty_object: {}

crazy_object: {
    key: variable
    key_two: "string here"
    key_three: [1 2 3]
    key_four: {
        nested_one: "alright"
        nested_two: [4 5 6]
    }
}
