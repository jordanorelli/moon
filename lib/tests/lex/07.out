{t_name empty_object}
{t_object_separator :}
{t_object_start {}
{t_object_end }}
{t_name crazy_object}
{t_object_separator :}
{t_object_start {}
{t_name key}
{t_object_separator :}
{t_name variable}
{t_name key_two}
{t_object_separator :}
{t_string string here}
{t_name key_three}
{t_object_separator :}
{t_list_start [}
{t_real_number 1}
{t_real_number 2}
{t_real_number 3}
{t_list_end ]}
{t_name key_four}
{t_object_separator :}
{t_object_start {}
{t_name nested_one}
{t_object_separator :}
{t_string alright}
{t_name nested_two}
{t_object_separator :}
{t_list_start [}
{t_real_number 4}
{t_real_number 5}
{t_real_number 6}
{t_list_end ]}
{t_object_end }}
{t_object_end }}
